src/js/views/globals/search/patient-search_views.js

```diff
@@ -42,11 +42,34 @@ const PatientSearchPicklist = Picklist.extend({
   itemTemplate: hbs`
     {{matchText text search}}{{~ remove_whitespace ~}}
     <span class="patient-search__picklist-item-meta">{{formatDateTime birth_date "MM/DD/YYYY"}}</span>
+    {{#each identifiers}}<span class="patient-search__picklist-item-meta">{{this}}</span>{{/each}}
   `,
   itemTemplateContext() {
+    // REMOVE: for sandbox testing purposes only
+    // this.model.set(
+    //  'identifiers',
+    //  [{ 'type': 'PID', 'value': '0123456789' }, { 'type': 'SSN', 'value': '321-45-6789' }, { 'type': 'XID', 'value': '9876543210' }],
+    // );
+
+    const settings = this.state.get('settings');
+    const settingsIdentifiers = settings ? settings.result_identifiers : [];
+
+    const patientIdentifiers = this.model.get('identifiers');
+
+    const identifiers = [];
+
+    each(settingsIdentifiers, settingsIdentifier => {
+      const identifierToShow = find(patientIdentifiers, patientIdentifier => {
```

Comment:
I think we can ensure that the type and settings are the same case.  `find(patientIdentifiers, { type: settingsIdentifier })` should be ok.